File: packages/dd-trace/test/llmobs/parity/scenarios/openai-agents/handoffs.js
'use strict'

const { researchWorkflow, setup } = require('./common')

async function main () {
  const { agents, finish } = setup()
  await agents.run(researchWorkflow(), 'What is a brief summary of what happened yesterday in the soccer world??')
  await finish()
}

main().catch(error => {
  process.stderr.write(`${error.stack ?? error}\n`)
  process.exitCode = 1
})

File: packages/dd-trace/test/llmobs/parity/scenarios/openai-agents/handoffs.py
import asyncio

from agents import Runner

from _agents_common import finish
from _agents_common import research_workflow
from _agents_common import setup


async def main():
    setup()
    await Runner.run(
        research_workflow(),
        "What is a brief summary of what happened yesterday in the soccer world??",
    )
    finish()


asyncio.run(main())
